src/main/java/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -500,6 +509,11 @@ private boolean shouldEnableCompleteExternalIdAccount(StudyParticipant participa
         return participant.getEmail() == null && participant.getPhone() == null && 
             participant.getExternalId() != null && participant.getPassword() != null;
     }
+    
+    private boolean shouldEnableCompleteSynapseUserIdAccount(StudyParticipant participant) {
+        return participant.getEmail() == null && participant.getPhone() == null && 
+            participant.getSynapseUserId() != null && participant.getPassword() == null;
+    }
```

Comment:
Is there any reason to gate this on email, phone, or password? If they have a Synapse User ID, even if they have a email, phone, or password, is there any reason we need to block them from signing in via Synapse?

Reply:
I think part of the confusion is that it’s not clear what the AccountStatus means. Back when Email was the only way to access Bridge, it was to prevent you from using Bridge before you verify your email, because a lot of things could use your email address. Now that we’ve split off email verification, the semantics is a little bit unclear.

From a purely technical perspective, UNVERIFIED means you can’t access Bridge until you’ve verified your email/phone number. If someone, for whatever reason, creates a Bridge account with both an email address and a Synpase ID, do we want to block them from using Bridge until they’ve verified that email address? Or do we want to allow them access to Bridge, but not send them any emails until they’ve verified their email address?

Reply:
Filed as https://sagebionetworks.jira.com/browse/BRIDGE-2628. I'll email Bridge group about this change today.